src/classes/AvoidDeeplyNestedIfStmts - Copy.cls

```diff
@@ -0,0 +1,13 @@
+public with sharing class AvoidDeeplyNestedIfStmts {
+
+	public void coords(Integer x, Integer y, Integer z) {
+		if(x>y) {
+			if(y>z) {
+				if(z==x) {
```

Comment:
Codacy found an issue: [Deeply nested if..then statements are hard to read](https://app.codacy.com/manual/mani2246/apex-pmd-samples/pullRequest?prid=5694177)